.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+**What is the problem this PR is trying to solve?**
```

Comment:
We can go for this template at first.
It's maybe a time a bit after to rethink if we want to change some stuff 🙂